src/util/ConfigManager/ConfigOption.h

```diff
@@ -26,7 +26,16 @@
 #include "util/json.h"
 
 namespace ad_utility {
-namespace innerNamespace {
+// Forward declarations needed for testing.
+namespace ConfigManagerImpl {
+template <typename... Ts>
+std::string generateValidatorName(std::optional<size_t> id);
+class ConfigManagerTest_AddNonExceptionValidator_Test;
+class ConfigManagerTest_AddExceptionValidator_Test;
+class ConfigManagerTest_PrintConfigurationDocComparison_Test;
+}  // namespace ConfigManagerImpl
+
+namespace ConfigOptionImpl {
```

Comment:
Also if you use a single namespace for both, then this stuff here all goes away again.

Reply:
Done, see [bd4eb95](https://github.com/ad-freiburg/qlever/pull/1214/commits/bd4eb9577b5b2f92aec344c88d0a6f8d44f8b5aa).